blog-posts/gatsby-ssr-caching.md

```diff
@@ -0,0 +1,123 @@
+---
+path: "/blog/cache-gatsby-ssr-pages-on-gatsby-cloud/"
+date: "2022-02-15"
+title: "Make Gatsby SSR blazing fast with caching on Gatsby Cloud"
+featuredImage: images/gatsby-ssr-caching.jpg
+author: "Fabian Dietenberger"
+authorSummary: "Senior Developer at Satellytes"
+teaserText: Learn how to cache Gatsby SSR pages on Gatsby Cloud
+seoMetaText: Learn how to cache Gatsby SSR pages on Gatsby Cloud to boost page speed and scalability. Your users won't see a difference to SSG or DSG pages anymore.
+leadboxText: Want to learn more about creating blazing fast pages? Then join us to learn more!
+attribution:
+    creator: Shiro Hatori
+    source: https://unsplash.com/photos/WR-ifjFy4CI
+---
+
+Ever since it's introduction [Gatsby](https://gatsbyjs.com) has established itself as the standard solution for 
+React-based [Jamstack](https://jamstack.org/what-is-jamstack/) architectures. But where this approach has proven to 
+be quite powerful for smaller sites, the restriction to pre-build all pages using Static Site Generation has proven 
+to be a bottleneck [for larger websites](https://www.gatsbyjs.com/docs/how-to/performance/improving-build-performance/)
+or pages with real-time requirements.
+
+Luckily the recent v4 update has added a couple of interesting new tools to the developer toolbox. One of them is 
+[Server-side Rendering](https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/). With 
+SSR, pages are created at the moment the user visits the page and not before. This allows for greater scaling and also 
+some additional features like per-user generated pages, but this is another blog post.
+
+## Why is caching important?
+
+When serving a page via SSR, efficient caching becomes crucial as the user won't see anything until the whole page 
+is rendered on the server and delivered to the browser. If rendering also includes fetching some external data or 
+doing some compute heavy calculations, it can also become problematic for scaling and stability. That's where caching 
+comes in place. You can cache on multiple levels, like only heavy computations. But the most impact on speed and 
```

Comment:
```suggestion
comes in. Caching can be done on multiple levels. But the most impact on speed and 
```

I would do a bullet list with the levels, e.g.: 
* caching on edge (*)
* caching render results (as static pages)
* caching database requests
* caching resources

(*) probably with link to definition)